lib/client.js

```diff
@@ -212,6 +212,7 @@ Client.prototype._setSequenceArgs = function(argsScheme, args) {
     return args;
   }
   for (var partIndex in argsScheme) {
+    partIndex = partIndex.replace('[]','')
```

Comment:
I tried to use with the brackets, but I had an error with parse response:

>     Error: Error: Cannot parse response
      at kplClient.CadastrarCliente (src/connectors/applications/kpl/services/kplService.js:35:25)
      at node_modules/soap/lib/client.js:180:7
      at finish (node_modules/soap/lib/client.js:451:14)
      at parseSync (node_modules/soap/lib/client.js:436:12)
      at node_modules/soap/lib/client.js:412:14
      at Request._callback (node_modules/soap/lib/http.js:118:5)
      at Request.self.callback (node_modules/request/request.js:188:22)
      at Request.<anonymous> (node_modules/request/request.js:1171:10)
      at IncomingMessage.<anonymous> (node_modules/request/request.js:1091:12)
      at endReadableNT (_stream_readable.js:975:12)
      at _combinedTickCallback (internal/process/next_tick.js:80:11)
      at process._tickCallback (internal/process/next_tick.js:104:9)